packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/WorklistDataGrid.1/About.2.doc.mdx

```diff
@@ -14,14 +12,102 @@ import WorklistDataGridColumnStates from './Examples.3/WorklistDataGridColumnSta
 Terra Data Grid is currently under alpha release. There may be breaking changes between releases until it is stable with the 1.0 release.
 </Notice>
 
+## Table of Contents
+* [Overview](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#overview)
+* [Design Decisions](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#design-decisions)
+* [Examples](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#examples)
+* [Accessibility](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#accessibility)
+* [Implementation](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#implementation)
+* [Terra Standards](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#terra-standards)
+* [Properties](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#properties)
+
+## Overview
```

Comment:
I noticed that the documentation template has changed from when we started with new suggestions. While the template is being finalized, I'm choosing to keep the examples as is for now as it does not make sense to have them in 2 places - Overview & Examples section.

Reply:
Sorry, I meant move the default example from the Examples to the Overview section. This is so that we have a visual when introducing the component and consumers do not have to scroll to see that the component.